src/components/FeatureViews/Registration/RegistrationCourseEnrollments.js

```diff
@@ -37,12 +48,28 @@ const TableToolbar = (
 );
 
 const RegistrationCourseEnrollments = ({courseID}) => {
+    const dispatch = useDispatch();
+    const api = useMemo(
+        () => ({
+            ...bindActionCreators(registrationActions, dispatch),
+        }),
```

Comment:
Since you're only using one `bindActionCreators`, you can rewrite this first parameter as:

     () => bindActionCreators(registrationActions, dispatch)